src/lib/pubsub/pubsub_build.c

```diff
@@ -254,13 +254,29 @@ dispatch_fill_pub_binding_backptrs(pubsub_builder_t *builder,
   } SMARTLIST_FOREACH_END(cfg);
 }
 
+/**
+ * Remove the dispatch_ptr fields for all the relevant publish bindings
+ * in <b>items</b>.  The prevents subsequent dispatch_pub_() calls from
+ * sending messages to a dispatcher that has been freed.
+ **/
+void
+pubsub_items_clear_bindings(pubsub_items_t *items)
```

Comment:
Seems to be untested?

Reply:
Now tested by 277fe13952de33f6f24be5c5ace64e817610b405